pkg/ddl/ddl.go

```diff
@@ -78,7 +79,8 @@ const (
 	// currentVersion is for all new DDL jobs.
 	currentVersion = 1
 	// DDLOwnerKey is the ddl owner path that is saved to etcd, and it's exported for testing.
-	DDLOwnerKey = "/tidb/ddl/fg/owner"
+	DDLOwnerKey         = "/tidb/ddl/fg/owner"
+	ddlSchemaVersionKey = "/tidb/schema_version"
```

Comment:
It is confused to have both `DDLAllSchemaVersions` and this. Maybe change to `ddlSchemaVersionLock`?